### PR TITLE
Setting templates

### PR DESCRIPTION
**Pull request by LizzAlice**

### CODE REVIEW

File: Lagerregal/template_production.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        'PORT': '',                      # Set to empty string for default. Not used with sqlite3.,
+    }
+}
+
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+# Example: "/home/media/media.lawrence.com/media/"
+MEDIA_ROOT =  ''
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = ""
```
Comment (octomike) [inline Lagerregal/template_production.py]: `None` didn't work? Or does `""` have the same effect?

File: Lagerregal/template_production.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    "main_department": "distinguishedName",
+    "expiration_date" : "accountExpires" # FIXME
+    }
+
+AUTH_LDAP_USER_FLAGS_BY_GROUP = {}
+
+AUTH_LDAP_DEPARTMENT_FIELD = "distinguishedName"
+AUTH_LDAP_DEPARTMENT_REGEX = "OU=([^,]*?),DC"
+
+#or false, if you don't want to use it
```
Comment (octomike) [inline Lagerregal/template_production.py]: redundant comment is redundant :)

File: Lagerregal/template_production.py (hunk 1 above)
Comment (LizzAlice) [reply Lagerregal/template_production.py]: I just forgot  :/

File: Lagerregal/template_production.py (hunk 2 above)
Comment (LizzAlice) [reply Lagerregal/template_production.py]: xD

**Pull request merged**